server/models/Request.js

```diff
@@ -0,0 +1,24 @@
+import mongoose from "mongoose";
+
+const { Schema, model } = mongoose;
+
+const requestSchema = new mongoose.Schema(
+  {
+    customer: { type: Schema.Types.ObjectId, ref: "User", required: true },
+    provider: { type: Schema.Types.ObjectId, ref: "User", required: true },
```

Comment:
These are nice! I was wondering maybe `owner/sitter` could be clearer names for them? Any thoughts?